docs/doxygen/overviews/helloworld.h

```diff
@@ -83,59 +84,46 @@ enum
 };
 @endcode
 
-Notice that you don't need to define identifiers for the "About" and "Exit". We
-then proceed to actually implement an event table in which the events are
-routed to their respective handler functions in the class MyFrame.
-
-There are predefined macros for routing all common events, ranging from the
-selection of a list box entry to a resize event when a user resizes a window on
-the screen. If @c wxID_ANY is given as the ID, the given handler will be
-invoked for any event of the specified type, so that you could add just one
-entry in the event table for all menu commands or all button commands etc.
-
-The origin of the event can still be distinguished in the event handler as the
-(only) parameter in an event handler is a reference to a wxEvent object, which
-holds various information about the event (such as the ID of and a pointer to
-the class, which emitted the event).
-
-@code
-wxBEGIN_EVENT_TABLE(MyFrame, wxFrame)
-    EVT_MENU(ID_Hello,   MyFrame::OnHello)
-    EVT_MENU(wxID_EXIT,  MyFrame::OnExit)
-    EVT_MENU(wxID_ABOUT, MyFrame::OnAbout)
-wxEND_EVENT_TABLE()
-@endcode
+Notice that you don't need to define identifiers for the "About" and "Exit" as
+wxWidgets already predefines and the standard values such as wxID_ABOUT and
+wxID_EXIT should be used whenever possible, as they can be handled in a special
+way by a particular platform.
 
 As in all programs there must be a "main" function. Under wxWidgets main is
-implemented using this macro, which creates an application instance and starts
-the program.
+implemented inside ::wxIMPLEMENT_APP() macro, which creates an application
+instance of the specified class and starts running the GUI event loop. It is
+used simply as:
 
 @code
-wxIMPLEMENT_APP(MyApp)
+wxIMPLEMENT_APP(MyApp);
 @endcode
 
 As mentioned above, wxApp::OnInit() is called upon startup and should be used
 to initialize the program, maybe showing a "splash screen" and creating the
-main window (or several). The frame should get a title bar text ("Hello World")
-and a position and start-up size. One frame can also be declared to be the top
-window. Returning @true indicates a successful initialization.
+main window (or several). As frames are created hidden by default, to allow
+creating their child windows before showing them, we also need to explicitly
+show it to make it appear on the screen. Finally, we return @true from this
+method to indicate successful initialization:
 
 @code
 bool MyApp::OnInit()
 {
-    MyFrame *frame = new MyFrame( "Hello World", wxPoint(50, 50), wxSize(450, 340) );
+    MyFrame *frame = new MyFrame();
     frame->Show( true );
     return true;
 }
 @endcode
 
 In the constructor of the main window (or later on) we create a menu with our
 menu items as well as a status bar to be shown at the bottom of the main
-window. Both have to be associated with the frame with respective calls.
+window. Both have to be associated with the frame with respective calls. We
```

Comment:
Suggest to start "We ..." as a new paragraph to help readability when adding the text I suggest below.